src/condor_tools/htcondor_cli/access_point.py

```diff
@@ -0,0 +1,89 @@
+import htcondor2
+from htcondor._utils.ansi import underline
+
+from htcondor_cli.convert_ad import ads_to_daemon_status, health_str
+from htcondor_cli.noun import Noun
+from htcondor_cli.verb import Verb
+
+
+def iter_list(l: list):
+    """Recursively yield each item in a list of lists"""
+    for item in l:
+        if type(item) is list:
+            yield from iter_list(item)
+        else:
+            yield item
+
+
+class Status(Verb):
+    """
+    Displays status information all/specified access points
+    """
+
+    options = {
+        "hostnames": {
+            "args": ("hostname",),
+            "nargs": "*",
+            "action": "append",
+            "help": "Specific Access Point hostnames to query",
+        }
+    }
+
+    def __init__(self, logger, **options):
+        hostnames = [host for host in iter_list(options["hostname"])]
+        constraint = None
+        if len(hostnames) > 0:
+            constraint = " || ".join(f'Machine == "{host}"' for host in hostnames)
+
+        collector = htcondor2.Collector()
+        ads = collector.query(ad_type=htcondor2.AdType.Schedd, constraint=constraint)
+
+        if len(ads) == 0:
+            raise RuntimeError("Failed to locate any Access Points from Collector")
+
+        ap_status = {}
+        width = 0
+
+        for daemon, status in ads_to_daemon_status(ads):
+            assert daemon == "SCHEDD"
+            host = status["Machine"]
+            width = len(host) if len(host) > width else width
+
+            if host in hostnames:
+                hostnames.remove(host)
+
+            if host in ap_status:
+                ap_status[host]["HP"] += status["HealthPoints"]
+                ap_status[host]["Count"] += 1
```

Comment:
Do we really expect more than one schedd per host?  Maybe suppress the count if it's 1?